plugins/katello/3.13/advanced/http_proxies.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: plugins/katello/documentation
+title: HTTP Proxies
+version: 'nightly'
+---
+
+# HTTP Proxies
+
+## HTTP Proxy Support
+
+Katello enables external HTTP proxies (provided by utilities such as [squid](http://www.squid-cache.org/)) for repository operations such as synchronization.
+
+HTTP proxies can be created and then assigned to a product though bulk selection as well as for each individual repository. Additionally, Katello provides HTTP proxy policies for products or repositories. Policies include:
+  * Using the global HTTP proxy (the default)
+  * Using a specified HTTP proxy other than the global HTTP proxy
+  * Not using an HTTP proxy
+
+## Creation
+
+There are two ways of creating a HTTP Proxy for use in Katello: through a rake task or the Foreman UI.
+
+### Creating an HTTP Proxy with Rake
+
+```
+foreman-rake katello:update_default_http_proxy -- --name "A Great HTTP Proxy" --url "http://192.0.2.1:8888" --user admin --password redhat
+```
```

Comment:
I wouldn't recommend the rake task at this point, originally the installer was going to be calling the rake task, but now the installer is just setting a config and db:seed is configuring it.  So i'd probably recommend just using the installer for now: 

```
--katello-proxy-password Proxy password for authentication (default: nil)
--katello-proxy-port Port the proxy is running on (default: nil)
--katello-proxy-url URL of the proxy server (default: nil)
--katello-proxy-username Proxy username for authentication (default: nil)
```